src/main/java/filter/expression/Qualifier.java

```diff
@@ -113,6 +113,64 @@ public static FilterExpression replaceMilestoneAliases(IModel model, FilterExpre
     }
 
     /**
+     * Expands user input to a matching keyword, returns original input otherwise
+     * Includes aliases for Qualifier that also functions as keyword 
+     * @param input
+     * @return 
+     */
+    public static String replaceKeywordAliases(String input) {
```

Comment:
Maybe this could be: "expands aliases for qualifier keywords in user input"? The verb in the method name could be changed to "expand" also. Feels like a more high-level name.